src/PantherTestCaseTrait.php

```diff
@@ -185,7 +185,9 @@ protected static function createPantherClient(array $options = [], array $kernel
 
         if (PantherTestCase::FIREFOX === $browser) {
             self::$pantherClients[0] = self::$pantherClient = Client::createFirefoxClient(null, $browserArguments, $managerOptions, self::$baseUri);
-        } else {
+        } elseif (PantherTestCase::SELENIUM === $browser) {
+            self::$pantherClients[0] = self::$pantherClient = Client::createSeleniumClient($managerOptions['host'], $managerOptions['capabilities'], self::$baseUri, $options);
```

Comment:
```suggestion
            self::$pantherClients[0] = self::$pantherClient = Client::createSeleniumClient($managerOptions['host'], $managerOptions['capabilities'] ?? null, self::$baseUri, $options);
```

Or you need to validate the manager options to enforce the key is defined.

and for `host`, you probably need to validate it as well.

Also, passing `$options` looks inconsistent with the case of other factories that are passing `$managerOptions` instead.